src/pluto_codegen_if.c

```diff
@@ -286,7 +318,13 @@ int pluto_gen_cloog_code(const PlutoProg *prog, int cloogf, int cloogl,
 
     cloogOptions->name = "PLUTO-produced CLooG file";
 
-    fprintf(outfp, "/* Start of CLooG code */\n");
+    if (prog->language == CLOOG_LANGUAGE_FORTRAN)
+        fprintf(outfp, "! Start of CLooG code\n");
+    else if (prog->language == CLOOG_LANGUAGE_PYTHON)
+        fprintf(outfp, "# Start of CLooG code\n");
+    else 
+        fprintf(outfp, "/* Start of CLooG code */\n");
```

Comment:
Please use braces around if the else/then block isn't on the same line.
else{
        ...
}

Reply:
Fixed this case and many others. Fixed `}else{` spaces too.